.github/workflows/tests_cpu.yml

```diff
@@ -0,0 +1,41 @@
+on: [push, pull_request]
+
+jobs:
+  test-cpu:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8]
+        pytorch-channel: [pytorch, pytorch-nightly]
```

Comment:
Not sure of the intention? Why do you want to test against the nightlies?

Reply:
so that we can have an idea of  our branch against future pytorch releases. Otherwise it's a surprise every time they release.

Reply:
I don't recommend that. The nightlies can be unstable and you might get failing PRs for reasons that you don't care about or you can't control. Furthermore, the nightlies, as the name implies, are updated daily and thus a lot of change can happen that you don't need to react to right away.

Here is what I would do: if a new PyTorch pre-release is announced, i.e. `torch==1.7.0.rc1`, you test `kornia` against this. Maybe create a `test-against-nightlies` workflow that is only triggered by pushes / PRs against a certain branch.

Reply:
what mechanism could we have to not have to update by hand pre-release names ? that's why I chose nigthlies, which btw seems we are doing fine